src/tests/ftest/container/open.py

```diff
@@ -34,16 +34,18 @@ class OpenContainerTest(TestWithServers):
     """
 
     def test_container_open(self):
-        """JIRA ID: DAOS-3223
+        """JIRA ID: DAOS-9965
```

Comment:
Does it make sense to keep both for reference/changelog?
```suggestion
        """JIRA ID: DAOS-3223
           JIRA ID: DAOS-9965
```

Reply:
Done